### DISCUSSION TITLE
Great job

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [epfl-SDP/android#445] [General]
Hi,

I came across your repo by chance and found it very interesting.
I'm one the ArSceneView developers and really appreciate seeing projects like your.

Do you have any screenshot or video record of your app to see what it looks like?

Congrats for your job

**Comment by KurohanaJuri** [comment]
Thank you for reaching out to us, we are happy to hear that! We had planned to write a short message on your Discord channel, but you are faster than us.

As written on the front page of this repo, this project is developed for the Software Development Project course at EPFL, and we were asked to produce a short video about the app. We can already give you this [video](https://www.youtube.com/watch?v=PIcgDgCp8y0&feature=youtu.be). If you want more footage or screenshots, don't hesitate to ask!

I would like to take the opportunity to thank you for your work on ArSceneView. You have made it easier to implement the AR feature in our app. I'm interested to take part in this project. I see that you want some help to write project samples, and it would be my pleasure to help you. Maybe we can discuss this on Discord ([My profile](https://discord.com/users/Juri#4072)) ?
